quickstarts/kubernetes-prometheus/config.yml

```diff
@@ -0,0 +1,75 @@
+slug: kubernetes-prometheus
+title: Kubernetes (Prometheus)
+description: |
+  ## What is Kubernetes?
+  Kubernetes is an open-source system for automating deployment, scaling, and, management of containerized applications. The New Relic Kubernetes (Prometheus) monitoring quickstart utilizes Prometheus to give you visibility into your Kubernetes clusters and workloads in minutes, whether your clusters are hosted on-premises or in the cloud.
+  
+  ## What is Prometheus? 
+  Prometheus is an open-source monitoring and alerting toolkit. 
+
+  ### Kubernetes (Prometheus) quickstart highlights
+  The New Relic Kubernetes quickstart uses dashboards to proactively monitor your metrics, like:
+  - read and write request metrics 
+  - memory and cpu utilization 
+  - running pods and containers 
+  - kubelet pod start metrics 
+  - kubelet storage operation metrics 
+  - compute resources by pod, node, cluster, and namespace and more. 
+
+  ### Requirements:
+  The approach to getting your Prometheus metrics into New Relic differs depending on which integration you use.
+
+  #### Prometheus Remote Write 
+  1. [Prometheus Operator](https://github.com/prometheus-operator/kube-prometheus)
+  2. Enable Prometheus [Remote Write](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-remote-write/set-your-prometheus-remote-write-integration/) to New Relic.
+  3. Try out the kube-prometheus stack with [minikube](https://github.com/prometheus-operator/kube-prometheus#minikube)
+
+  you will need to add the remote write configuration to your `manifests/prometheus-prometheus.yaml` config file:
+  `
+  remoteWrite:
+      - url: https://metric-api.newrelic.com/prometheus/v1/write?prometheus_server=<YOUR_SERVER_NAME>
+        authorization:
+          credentials:
+            key: value
+            name: nr-license-key
+  `
+  `<YOUR_SERVER_NAME>` is an attribute added to your data that will help you identify which prometheus server is sending data to New Relic. 
+
+  to create your nr-license-key secret for the config to read from, run the following command: 
+  `kubectl -n monitoring create secret generic nr-license-key --from-literal=value=<YOUR_NEW_RELIC_API_INGEST_KEY>` 
+
+  `<YOUR_NEW_RELIC_API_INGEST_KEY>` is an api key found in your New Relic account settings. This key MUST be an INGEST key. 
+summary: |
+  Monitoring Kubernetes is crucial to gain instant visibility into Kubernetes clusters and workloads. 
+  Prometheus' ability to monitor a wide variety of data necessitates monitoring the toolkit itself. New Relic helps store, manage, and view telemetry data from your Prometheus setup, relieving you of a significant operational burden.
+  Download New Relic Kubernetes (Prometheus) quickstart to proactively monitor Kubernetes cluster health and capacity.
+level: Community
+authors:
+  - New Relic 
+title: Kubernetes (Prometheus) 
```

Comment:
Looks like the validation is failing because there are 2 `title` keys in this file.
```suggestion
```

Reply:
ooh, missed that. thanks